CHANGELOG.md

```diff
@@ -32,10 +32,8 @@
 - Change `KafkaConnect` field `userConfig.additional_backup_regions`: deprecated
 - Change `Kafka` field `userConfig.additional_backup_regions`: deprecated
 - Change `OpenSearch` field `userConfig.additional_backup_regions`: deprecated
-- Change `PostgreSQL` field `userConfig.pg_version`: enum ~~`[11, 12, 13, 14, 15]`~~ → `[11, 12, 13,
-14, 15, 16]`
+  14, 15, 16]`
```

Comment:
This seems incorrect: 
![image](https://github.com/aiven/aiven-operator/assets/25483483/8c57e11f-2abc-4568-b8ac-c5e41950562a)

Reply:
Thanks. I regenerated from changelog since the last release.